service/src/test/resources/ehr/mapper/codeableconcept/codeable_concept_with_text.json

```diff
@@ -0,0 +1,35 @@
+{
+    "resourceType" : "Observation",
+    "code": {
+        "coding": [
+            {
+                "system": "http://blah.info/blahsomecodesystem",
+                "code": "X%^^%",
+                "display": "Endometriosis of uterus",
```

Comment:
there could be different display to verify if the taken display is really snomed code